### ISSUE TITLE
Support for type hints

### ISSUE THREAD

**Issue by lukaschoebel** [shivam5992/textstat#146]
In order to conform to [PEP484](https://www.python.org/dev/peps/pep-0484/), it is beneficial to explicitly type the arguments and return types of functions.

This issue might be lower priority but would help users to prevent pitfalls in the future.

**Comment by alxwrd** [comment]
Definitely. With Python 3.5 out of support now, I would expect all consumers of the library to be on a version of Python that supports type hinting.
